bin/junit/TEST-za.ac.sun.cs.green.EntireSuite.xml

```diff
@@ -0,0 +1,240 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Please make use of your .gitignore. 180 unnecessary files were added to this repo.